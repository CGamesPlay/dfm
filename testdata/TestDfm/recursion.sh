#!/bin/bash
# Tests that specific-path commands are working as intended.
set -e
. "$(dirname "$0")/../helpers.sh"

export HOME="$(pwd)/home"
export DFM_DIR="$HOME/dfmdir"

mkdir -p ~/dfmdir/files

dfm init --repos files
cd ~/dfmdir/files
echo 'vim config' > .vimrc
dfm add .vimrc && fail 'file recursively added' || true
dfm link .vimrc
