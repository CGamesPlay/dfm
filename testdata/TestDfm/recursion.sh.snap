$ dfm init --repos files
Initialized /test/home/dfmdir as a dfm directory.
$ dfm add .vimrc
dfmdir/files/.vimrc: cannot add a file already inside the dfm directory
$ dfm link .vimrc
files/.vimrc -> /test/home/.vimrc
